Curve-finance-suite/DAOKillingGauge/src/agent.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { HandleTransaction } from "forta-agent";
+import { TestTransactionEvent, createAddress } from "forta-agent-tools";
+import { providerHandleTransaction, createFinding } from "./agent";
+import { gaugeInterface } from "./abi";
+
+const curveDAOAddress = createAddress("0x1");
+
+describe("Curve DAO Kill a Gauge Test Suite", () => {
+  let handleTransaction: HandleTransaction;
+
+  beforeAll(() => {
+    handleTransaction = providerHandleTransaction(curveDAOAddress);
+  });
+
+  it("should return empty findings method was not called", async () => {
```

Comment:
i think this can be improved,
 'Should return empty findings when set_killed was not called' or
'Should return empty findings with empty transactions' for a more specific description.